src/datagrid/DataGrid.tsx

```diff
@@ -701,25 +706,36 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
         const {selectionType} = this.props;
         const {selectAll} = this.state;
         return (
-            <div
-                role="columnheader"
-                className={classNames([
-                    ClassNames.DATAGRID_COLUMN, //prettier
-                    ClassNames.DATAGRID_SELECT,
-                    ClassNames.DATAGRID_FIXED_COLUMN,
-                ])}
-            >
-                <span className={ClassNames.DATAGRID_COLUMN_TITLE}>
-                    {selectionType === GridSelectionType.MULTI && (
-                        <CheckBox
-                            id="select_all"
-                            onChange={evt => this.handleSelectAll(evt)}
-                            ariaLabel="Select All"
-                            checked={selectAll !== undefined ? selectAll : undefined}
-                        />
-                    )}
-                </span>
-                <div className={ClassNames.DATAGRID_COLUMN_SEPARATOR} />
+            <div className={ClassNames.DATAGRID_ROW_STICKY}>
+                <div
+                    role="columnheader"
```

Comment:
Can we move these hardcoded values to constants or enum.

Reply:
@maheshkulkarni4 those are part of DOM there is no use of this in code logic so not converting it to enum. Clarity may have some CSS associated with it.